packages/superset-ui-legacy-plugin-chart-sunburst/src/utils/wrapSvgText.js

```diff
@@ -29,8 +29,8 @@ export default function wrapSvgText(text, width, adjustedY) {
       .attr('dy', `${dy}em`);
 
     let didWrap = false;
-    for (let i = 0; i < words.length; i++) {
-      word = words[i];
+    words.forEach(element => {
```

Comment:
this is fine, but you could potentially not declare `word` above, and just `forEach(word => ...)` directly